sideboard/server.py

```diff
@@ -125,7 +126,7 @@ class SideboardWebSocket(WebSocketDispatcher):
     @classmethod
     def check_authentication(cls):
         host, origin = cherrypy.request.headers['host'], cherrypy.request.headers['origin']
-        if ('//' + host) not in origin:
+        if ('//' + host.split(':')[0]) not in origin:
```

Comment:
Accounts for nonstandard ports.  This is only really useful in dev mode or weird configurations.